### PR TITLE
Installation guide refactor

### PR DESCRIPTION
**Pull request by Ieno77**
# Description
Update doc screenshots

# Impacts
doc

### CODE REVIEW

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 1]
```diff
@@ -1037,9 +1037,12 @@ Where :
 * *Workgroup* is the workgroup of your domain in the old syntax (like NT4).
 * *DNS name of the domain* is the FQDN of your domain. The one that suffixes your account names.
 * *This server's name* is the name that the server's account will have in your Active Directory.
+* *Sticky DC* is the prefered domain controller to connect to.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: preferred

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 2]
```diff
@@ -1037,9 +1037,12 @@ Where :
...
+* *Active Directory server* is the IP or DNS name of one of the DC of the domain.
 * *DNS server* is the IP address of the DNS server of this domain. Make sure that the server you put there has the proper DNS entries for this domain.
-* *Username* is the username that will be used for binding to the server. This account must be a domain administrator.
-* *Password* is the password for the username defined above.
+* *OU* is the OU in the Active Directory where you want to create your computer account.
+* *ntlmv2 only* forces the NTLNM authentication (802.1x on AD) to use the NTLM version 2.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: 802.1X

File: docs/PacketFence_Installation_Guide.asciidoc (hunk 1 above)
Comment (julsemaan) [reply docs/PacketFence_Installation_Guide.asciidoc]: Also, please adjust this list to be a numbered list. Ensure you follow proper standards while doing so

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 3]
```diff
@@ -1070,17 +1073,34 @@ First configure your domains in 'Configuration -> Policies and Access Control ->
...
+* *Type* is the home server pool type
+* *Eduroam Realm Options* You can add Eduroam FreeRADIUS options in the realm definition
+* *Eduroam RADIUS Auth* is the RADIUS Eduroam authentication server to proxy the request to
+* *Type* is the home server pool type
+* *Authorize from PacketFence* specifies if we forward the request to PacketFence to have a dynamic answer or do we use the remote proxy server answered attributes?
+* *Eduroam RADIUS Acct* is the RADIUS Eduroam accounting server to proxy the request to
+* *Type* is the home server pool type
+* *Strip on the portal* Should the usernames matching this realm be stripped when used on the captive portal
+* *Strip on the admin* Should the usernames matching this realm be stripped when used on the administration interface
+* *Strip in RADIUS authorization* Should the usernames matching this realm be stripped when used in the authorization phase of 802.1x
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: 802.1X

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 4]
```diff
@@ -1070,17 +1073,34 @@ First configure your domains in 'Configuration -> Policies and Access Control ->
...
+* *Custom attributes* Allow to use custom attributes to authenticate 802.1x users (attributes are defined in the source)
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: 802.1X

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 5]
```diff
@@ -1070,17 +1073,34 @@ First configure your domains in 'Configuration -> Policies and Access Control ->
...
 
 * *Realm* is either the DNS name (FQDN) of your domain or the workgroup
+* *Domain* is the Active Directory domain where PacketFence sends the NTLM request
 * *Realm options* are any realm options that you want to add to the FreeRADIUS configuration
 * *Domain* is the domain which is associated to this realm
-
-*Now create the two other realms associated to your other domains.*
+* *RADIUS Auth* is the RADIUS authentication server to proxy the request to
+* *Type* is the home server pool type
+* *Authorize from PacketFence* specifies if we forward the request to PacketFence to have a dynamic answer or do we use the remote proxy server answered attributes?
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: remove the question mark

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 6]
```diff
@@ -1070,17 +1073,34 @@ First configure your domains in 'Configuration -> Policies and Access Control ->
...
+* *RADIUS Acct* is the RADIUS accounting server to proxy the request to
+* *Type* is the home server pool type
+* *Eduroam Realm Options* You can add Eduroam FreeRADIUS options in the realm definition
+* *Eduroam RADIUS Auth* is the RADIUS Eduroam authentication server to proxy the request to
+* *Type* is the home server pool type
+* *Authorize from PacketFence* specifies if we forward the request to PacketFence to have a dynamic answer or do we use the remote proxy server answered attributes?
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: remove the question mark

**Pull request merged**